.travis.yml

```diff
@@ -80,9 +82,13 @@ matrix:
     - python: "2.7"
       env: TOX_ENV=flake8
     - python: "3.5"
-      env: TOX_ENV=py27-backend_postgres KUBERNETES_VERSION=v1.9.0
+      env: TOX_ENV=py27-backend_postgres KUBERNETES_VERSION=v1.9.0 GIT_SYNC=0
     - python: "2.7"
-      env: TOX_ENV=py35-backend_postgres KUBERNETES_VERSION=v1.10.0
+      env: TOX_ENV=py35-backend_postgres KUBERNETES_VERSION=v1.10.0 GIT_SYNC=0
+    - python: "3.5"
+      env: TOX_ENV=py27-backend_postgres KUBERNETES_VERSION=v1.9.0 GIT_SYNC=1
+    - python: "2.7"
+      env: TOX_ENV=py35-backend_postgres KUBERNETES_VERSION=v1.10.0 GIT_SYNC=1
```

Comment:
Does this have to be defined as a separate build env in travis? Is it possible to achieve it as just separate tests/config in the same built environment?

(This matrix is unwieldy enough as it is, so I'd like us to not have to add new combinations here if we can avoid it cleanly

Reply:
And error prone: This has duplicated a non kube env:

TOX_ENV=py27-backend_postgres KUBERNETES_VERSION=v1.9.0 GIT_SYNC=0
TOX_ENV=py27-backend_postgres KUBERNETES_VERSION=v1.9.0 GIT_SYNC=1